### PR TITLE
Update dev team section.

### LINKED ISSUE #951: About page dev component
### What's your idea?

Redesign the dev component in acmcsuf.com/about following the provided image. Two separate logos are provided to interchange when the semester button is changed. (Mobile mode structure is provided in separate issues. [Reference.](https://github.com/EthanThatOneKid/acmcsuf.com/issues/941)

Light mode desktop -F23/Light mode desktop F23+ 
image
Dark mode desktop -F23/Dark mode desktop F23+ 
image

### PR DESCRIPTION
**Pull request by 26samaahmed**
Screen Shot 2023-11-08 at 7 29 18 PM
Fix #951

### CODE REVIEW

File: src/routes/(site)/6/teams/team-section.svelte [hunk 1]
```diff
@@ -51,6 +55,22 @@
...
+
+  @keyframes spin {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
```
Comment (EthanThatOneKid) [inline src/routes/(site)/6/teams/team-section.svelte:72]: Let's disable the animations in the case that a user prefers reduced motion.[^1]

```suggestion
@media (prefers-reduced-motion: reduce) {
  .dev-animation img {
    animation: none;
  }
}
```

> The setting is used to convey to the browser on the device that the user prefers an interface that removes, reduces, or replaces motion-based animations.
> Such animations can trigger discomfort for those with [vestibular motion disorders](https://www.a11yproject.com/posts/understanding-vestibular-disorders/).[^2]

[^1]: https://web.dev/articles/prefers-reduced-motion#working_with_the_media_query_from_css_and_javascript_contexts
[^2]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Nice, [small PR](https://google.github.io/eng-practices/review/developer/small-cls.html)! Please see my comment for a suggestion regarding user preferences.

**Comment by 26samaahmed** [comment]
I added your suggestion!!

**Review by EthanThatOneKid — APPROVED** [review]
This PR looks good to go! Nice work, @26samaahmed.

**Pull request merged**